README.md

```diff
@@ -346,6 +346,18 @@ Symbiote\QueuedJobs\Services\QueuedJobService\QueuedJobsService:
 ALTER TABLE `QueuedJobDescriptor` ADD INDEX ( `JobStatus` , `JobType` )
 ```
 
+## Job error logging
+
+The logger can be attached to a helper which is executed within a job like below:
+
+```php
+$job = new YourQueuedJob($someArguments);
+$logger = new Logger();
+$logger->setJob($job);
+$helper = Helper::create();
+$helper->setLogger($logger);
```

Comment:
I think this example should be changed to reflect a more common situation where the logging is attached within the job execution. i.e. within the `process()` function of the job

Reply:
You mean something like this?
```
$logger = new Logger();
$logger->setJob($job);

$helper = Helper::create();
$helper->setLogger($logger)
            ->run();
```

Reply:
Something like this:

```
// within your job class

public function process(): void
{
    $logger = new Logger();
    $logger->setJob($this);

    Helper::create()
        ->setLogger($logger)
        ->run();
}
```

Reply:
Given that `Helper` isn't defined, I still think this is quite confusing. Regardless, I'm not sure why there's a need to mix in the concept of a "helper class" with the creation of a logger, seems like unnecessary noise? It's pretty clear how a logger can be used (incl. passing it to other hypothetical dependencies) once it's created.